src/api/app/models/bs_request_action.rb

```diff
@@ -768,7 +765,7 @@ def check_action_permission!(skip_source = nil)
       # validate group object
       Group.find_by_title!(group_name)
     end
-    if self.role
```

Comment:
This can't be changed, here `self.role` is an attribute, and role is a local variable to this method